mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (order) {
+      return res.status(404).json({ message: "Order does not exist" });
```

Comment:
The code snippet is returning a 404 status code with a JSON response indicating that the order does not exist. This is a good practice as it provides a clear error message to the client. However, it would be better to use a consistent error handling mechanism throughout the application. 

For example, in other parts of the code, the error handler function `errorHandler` is used to handle errors. It would be more consistent to use this function here as well, instead of returning a custom error message. 

Additionally, it would be a good idea to log the error on the server-side for debugging purposes. This can be done using a logging mechanism such as console.error or a logging library like winston.